demo/docker/grafana/dashboards/Flink-ESP.json

```diff
@@ -1429,6 +1429,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "influx",
+      "description": "IMPORTANT. \nThis metrics does NOT show the overall lag on Kafka source; a largest lag among all the source partitions is shown. ",
```

Comment:
"This metrics" -> "This metric", and while we're changing it we may remove unnecessary space at end :D